asreview/webapp/api/projects.py

```diff
@@ -810,6 +810,44 @@ def api_random_prior_papers(project):  # noqa: F401
                 }
             )
 
+    elif subset == "unseen":
+        # Fetch records that are unseen
+        unlabeled_indices = as_data.df[as_data.df["debug_label"] == -1].index.values
```

Comment:
```suggestion
        unlabeled_indices = as_data.df[as_data.df["debug_label"] == LABEL_NA].index.values
```

where `LABEL_NA` is imported from `asreview.config`  (`from asreview.config import LABEL_NA`)